src/utils/window-console.js

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import { stripIndent } from 'common-tags';
+import type { GetState } from '../types/store';
+import { getProfile, selectedThreadSelectors } from '../reducers/profile-view';
+
+// Provide a bit of typing hinting. Flow doesn't understand the dynamic nature of
+// Object.defineProperty and has no definitions around it.
+type DefineGetProperty<T> = (
+  object: Object,
+  key: string,
+  {
+    +get: () => T,
+  }
+) => void;
+
+/**
+ * This function adds various values from the Redux Store to the window object so that
+ * people can access useful things.
+ */
+export function addDataToWindowObject(getState: GetState) {
```

Comment:
nit: what do you think of passing the `window` as a parameter to the function? In `index` you'd pass it, but in the test you could just pass `{}` and directly compare it then.